tokio-util/src/lib.rs

```diff
@@ -55,6 +55,11 @@ pub mod sync;
 
 pub mod either;
 
+/// A partial reexport of the bytes crate.
+pub mod bytes {
+    pub use bytes::{Buf, BufMut, Bytes, BytesMut};
+}
```

Comment:
At this point it might be more reasonable to just re-export the entire crate.
```suggestion
/// A partial reexport of the bytes crate.
pub use bytes;
```